docs/en/backup-restore.rst

```diff
@@ -2,56 +2,158 @@
 
 .. _backup-restore.rst:
 
-backup-restore.rst
-+++++++++++++++++++++++++++
+Backup and Restore
+++++++++++++++++++
+The **Backup and Restore** functionality becomes relevant when the User can no longer access the Digital Credentials stored on the mobile device where the Wallet Instance was installed. Situations that may require the User to utilize this functionality include:
+
+- The mobile device has been **lost**, **stolen**, **damaged**, or **compromised** (e.g., due to unauthorized access).
+- The User replaces an existing Wallet Instance with a new instance of the same Wallet Solution.
+- The User has changed their mobile device and needs to set up the Wallet Solution on the new device.
+- The User performs a factory reset on the current phone and needs to set up the Wallet Solution again.
+
+.. note::
+  
+  The Backup and Restore functionality differs from migrating to another Wallet Solution (also known as Data Portability). In the latter scenario, the User wishes to transition from their current Wallet Solution to a different one, possibly because the current solution is no longer supported, as highlighted in Annex 2 of ARF.
```

Comment:
```suggestion
  The Backup and Restore functionality differs from migrating to another Wallet Solution (also known as Data Portability). In the latter scenario, the User wishes to transition from their current Wallet Solution to a different one, possibly because the current solution is no longer supported, as highlighted in Annex 2 of `EIDAS-ARF`_.
```